lib/rules/use-tokens/index.js

```diff
@@ -94,15 +104,16 @@ const rule = primaryOption => {
             return;
 
           // If its a word and the value does not match expected values we report and error
-          if (element.type === 'word' && !checkValue(element.value)) {
-            message = messages.expected(prop, element.type);
-          } else if (element.type === 'numeric') {
+          if (
+            (element.type === 'word' && !checkValue(element.value)) ||
+            element.type === 'numeric'
```

Comment:
Should we update the commend above to explain why numeric is always bad?

Reply:
Sure